integration/teleterm_test.go

```diff
@@ -317,8 +316,8 @@ func testHeadlessWatcher(t *testing.T, pack *dbhelpers.DatabasePack, creds *help
 	// Ensure the watcher catches events and sends them to the Electron App.
 
 	require.EventuallyWithT(t, func(collect *assert.CollectT) {
-		eventuallyCatchAndSendHeadlessAuthn(collect)
-	}, time.Second, 100*time.Millisecond)
+		eventuallyCatchAndSendHeadlessAuthn(t)
```

Comment:
I'm not sure I fully understand `EventuallyWithT`. Should we be using `collect` here?